src/DataDepsGenerators.jl

```diff
@@ -114,6 +114,11 @@ function format_papers(authors::Vector, year::String, name::String, link::String
     join(authors, ", ") * " ($year). " * name * " " * link
 end
 
+function check_dois(uri::String)
```

Comment:
`doi` singlular, as it is only one.

and `check` isn't right.
_Check_ implies that this returns `true` if something is a DOI, and false if not.

`extract`, `try_extract` or `match` (since this corresponds to the regex), are better.
Maybe `get_match`